Xamarin.Forms.Build.Tasks/xlf/ErrorMessages.cs.xlf

```diff
@@ -0,0 +1,182 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
+  <file datatype="xml" source-language="en" target-language="cs" original="../ErrorMessages.resx">
+    <body>
+      <trans-unit id="AdderMissing">
+        <source>No Add() method defined on "{0}{1}".</source>
+        <target state="new">No Add() method defined on "{0}{1}".</target>
+        <note>0 is type name, 1 is property name</note>
+      </trans-unit>
+      <trans-unit id="BPMissingGetter">
+        <source>Missing a "public static Get{0}" method or a public instance property getter for the attached property "{1}.{0}Property".</source>
+        <target state="new">Missing a "public static Get{0}" method or a public instance property getter for the attached property "{1}.{0}Property".</target>
+        <note>0 is BindableProperty name, 1 is the declaring type</note>
+      </trans-unit>
+      <trans-unit id="BPName">
+        <source>The name of the BindableProperty {0} does not end with "Property". This is the kind of convention the world is build upon, a bit like Planck's constant.</source>
```

Comment:
Can we delete `This is the kind of convention the world is build upon, a bit like Planck's constant.` out of all these files?